.travis.yml

```diff
@@ -1,22 +1,65 @@
-sudo : false
+sudo: false
 
 notifications:
   email: false
 
 language: C++
 
-compiler: 
-    - gcc
+matrix:
+  include:
+    - compiler: gcc
+      env: 
+        - COMPILE_WITH_MPI=OFF
+        - COMPILER=/usr/bin/g++
+    - compiler: gcc
+      env: 
+        - COMPILE_WITH_MPI=ON
+        - COMPILER=/usr/bin/mpicxx
```

Comment:
Could we make this one run with C++11 set to OFF? We keep accidentally introducing C++11isms.